mass2/core/multifit.py

```diff
@@ -194,6 +220,7 @@ def to_pfit_gain(self, previous_energy2ph: Callable) -> tuple[np.polynomial.Poly
         pfit_gain = np.polynomial.Polynomial.fit(peaks_uncalibrated, gain, deg=2)
 
         def ph2energy(ph: NDArray) -> NDArray:
+            "The gain calibration curve: ph -> energy"
```

Comment:
Given an array `ph` of pulse heights, return the corresponding energies as an array.